tests/hierarchical/hierarchical_non_uniform_local_range.cpp

```diff
@@ -50,6 +50,17 @@ void check_expected(const std::vector<sycl::int3> &data, unsigned local_id,
   }
 }
 
+// there is no id -> range conversion in the spec
+static sycl::range<1> id_to_range(sycl::id<1> id) {
+  return sycl::range<1>(id[0]);
```

Comment:
This is true but there is 
```c++
// only available if Dimensions == 1
operator size_t() const;
```
https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#id-class

Reply:
Yep, but this is a generic function and the case through size_t only works for Dims=1.